src/qml/QFieldCloudScreen.qml

```diff
@@ -554,6 +554,10 @@ Page {
             cloudProjectsModel.removeLocalProject(projectActions.projectId)
             iface.removeRecentProject(projectActions.projectLocalPath);
             welcomeScreen.model.reloadModel()
+
+            if (projectActions.projectLocalPath === qgisProject.fileName) {
+              qgisProject.fileName = ""
```

Comment:
This is a bit of a hack here, shouldn't we instead properly close the project?

Settings the fileName to "" edits the project, it doesn't clear it of its content.

Reply:
`QgsProject.clear()` is not invocable, so it needs a utility function. Will close this PR and come back to this when I have time.

Reply:
@suricactus , yeah, I was thinking you could add a iface.clearProject() , that'd be easy enough

Reply:
Nice reminder that we have this module. Seems I was not the only one to thing where to put some utility function :) .